### ISSUE TITLE
Tempo plugin: 500 error response when loading Search tab

### ISSUE THREAD

**Issue by gng0** [grafana/grafana#87201]
### What happened?

When loading the Search tab in the Grafana Tempo plugin, we are seeing 500 errors every time on page load. 

This is coming from the `/api/v2/search/tag/name/values` endpoint (loading the Span Name values dropdown), where the response size is exceeding the max. configured value. 

As this API endpoint doesn't support pagination, @joe-elliott has suggested using the new streaming tags endpoint https://github.com/grafana/tempo/blob/main/cmd/tempo-cli/cmd-query-search-tag-values.go which will be a more scalable solution for this issue.

Additional details can be found in https://github.com/grafana/tempo/issues/3599

### What did you expect to happen?

We expect no errors to be thrown when loading the Search tab

### Did this work before?

Not sure

### How do we reproduce it?

1. Open the Search tab on the Tempo plugin
2. If the size of all your span names exceeds your max configured value, you can see the 500 errors in DevTools

### Environment (with versions)?

Grafana:
OS:
Browser:

### Grafana platform?

None

**Comment by joe-elliott** [comment]
Tempo appears to be timing out when there are a very large number of span names. Supporting the streaming endpoint would help.

**Comment by joe-elliott** [comment]
Yup, I would recommend using `max_bytes_per_tag_values_query` in the overrides section:

https://grafana.com/docs/tempo/latest/configuration/#overrides
